composer.json

```diff
@@ -16,7 +16,7 @@
     }
   ],
   "require": {
-    "drupal/migrate_plus": "^4",
+    "drupal/migrate_plus": "*",
     "jonathangreen/tuque": "dev-master"
```

Comment:
Does this also need a requirement on [migrate_source_csv](https://www.drupal.org/project/migrate_source_csv) specifically ^3 given the syntax has changed?